mesonpy/__init__.py

```diff
@@ -542,7 +546,49 @@ def build(self, directory: Path) -> pathlib.Path:
         return wheel_file
 
     def build_editable(self, directory: Path) -> pathlib.Path:
-        self._build()
+        self._project.build()  # ensure project is built
+
+        wheel_file = pathlib.Path(directory, f'{self.name}.whl')
+
+        project_path = self._source_dir / '.mesonpy-editable'
+        top_level_modules = self._project.top_level_modules
+        rebuild_commands = [list(cmd) for cmd in self._project.build_commands(project_path)]
```

Comment:
Why the list comprehension?

Reply:
To convert to the correct type.

Reply:
Correct type for what? This makes tuples into lists, but it does not seem like lists are required. Are they? If they are, can't we just return the correct type?